pkgs/applications/audio/soundux/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib, stdenv, fetchFromGitHub, cmake, ninja, pkg-config, makeWrapper, wrapGAppsHook,
+  libappindicator-gtk3, openssl, pipewire, pulseaudio, webkitgtk, xorg,
+  libpulseaudio, libwnck3,
+  downloaderSupport ? true, ffmpeg, youtube-dl
+}:
+
+let
+  downloaderPath = lib.makeBinPath [ffmpeg youtube-dl ];
+  # Soundux loads pipewire, pulse and libwnck optionally during runtime
+  dynamicLibraries = lib.makeLibraryPath [libpulseaudio pipewire libwnck3 ];
+in
+stdenv.mkDerivation rec {
+  pname = "soundux";
+  version = "0.2.7";
+
+  src = fetchFromGitHub {
+    owner = "Soundux";
+    repo = "Soundux";
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "15kd9vl7inn8zm5cqzjkb6zb9xk2xxwpkm7fx1za3dy9m61sq839";
+  };
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
+
+  dontWrapGApps = true;
+
+  installPhase = ''
+    mkdir -p $out/opt $out/bin
+    cp -r dist soundux-${version} $out/opt
+  '';
+
```

Comment:
Would you please add `desktopItem` attribute here (check [discord package](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/discord/linux.nix#L97-L105) for example)

Reply:
You'll also need to create `share` directory and create a symlink to it like [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/discord/linux.nix#L94)